packages/driver/src/dom/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "globals": {
+    "Cypress": true
+  }
+}
```

Comment:
this is code smell needing to pull in globals - i get the challenge - you want DOM utils to be stateless, and avoid DI or instantiation but there's probably a middle ground here we can facilitate